utilities/input_tools.py

```diff
@@ -367,14 +367,14 @@ def safeOpenRootFile(fileName, quitOnFail=True, silent=False, mode="READ"):
     fileObject = ROOT.TFile.Open(fileName, mode)
     if not fileObject or fileObject.IsZombie():
         if not silent:
-            print(f"Error when opening file {fileName}")
+            logger.error(f"Error when opening file {fileName}")
         if quitOnFail:
             quit()
         else:
             return None
     elif not fileObject.IsOpen():
         if not silent:
-            print(f"File {fileName} was not opened")
+            logger.error(f"File {fileName} was not opened")
```

Comment:
it might be a bit nicer to do this with by raising an exception, then quitOnFail can decide if the exception is caught or not. That way if it does fail you get the stack trace from the exception